.github/workflows/test.yml

```
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ '10', '12', '14' ]
+        node: [ '12', '14', '16']
```

Node.jsの現在のLTS的に追従すべきで、14,16,18が良さそう
https://nodejs.org/en/about/releases/
```suggestion
        node: [ '14', '16', '18']
```